### PR TITLE
Android demo: Explicitly import R.java

### PR DESCRIPTION
**Pull request by andrewharp**
Add explicit import of the R class to fix Google-internal Android demo builds.

### CODE REVIEW

**Comment by andrewharp** [comment]
For context, this is necessary for internal Google builds as it makes it easier for us to change the application package.

**Comment by andrewharp** [comment]
@tensorflow-jenkins Test this, please.

File: tensorflow/examples/android/src/org/tensorflow/demo/ClassifierActivity.java [hunk 1]
```diff
@@ -36,6 +36,9 @@
 import org.tensorflow.demo.env.ImageUtils;
 import org.tensorflow.demo.env.Logger;
 
+// Eplicit import needed for internal Google builds.
```
Comment (vrv) [inline tensorflow/examples/android/src/org/tensorflow/demo/ClassifierActivity.java]: spelling

**Comment by vrv** [comment]
@tensorflow-jenkins test this please

**Review by vrv — APPROVED** [review]

**Pull request merged**